### PR TITLE
Add HW12

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW12/test-hw12.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "Sid": "Stmt1537365816157",
+      "Action": [
+        "s3:CreateBucket"
+      ],
+      "Effect": "Deny",
+      "Resource": [
+        "arn:aws:s3:::test1",
+        "arn:aws:s3:::backup-test*"
+ "arn:aws:s3:::hw12-bucket"
```
Comment (mavrik90) [inline HW12/test-hw12.json]: should be tabs here

File: HW12/task4.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+tput setaf 4; echo "Enter the number B"; tput setaf 7 
+read B
+if [[ $B == "exit" ]] || [[ $B == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+tput setaf 4; echo "Enter the number C"; tput setaf 7 
+read C
+if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
```
Comment (mavrik90) [inline HW12/task4.sh:23]: Please add A and C compare logic.

Comment (mavrik90) [reply HW12/task4.sh:23]: The rest looks cool!

File: HW12/test-hw12.json (hunk 1 above)
Comment (Gritskov-Vitaliy) [reply HW12/test-hw12.json]: Done!
image

File: HW12/task4.sh (hunk 2 above)
Comment (Gritskov-Vitaliy) [reply HW12/task4.sh:23]: Done!

**Comment by Gritskov-Vitaliy** [comment]
Added checking for characters other than e and exit, and error output

image
image

**Review by mavrik90 — APPROVED** [review]
Nice!

**Pull request merged**